### ISSUE TITLE
order of operations

### ISSUE THREAD

**Issue by paulle** [SimpleMobileTools/Simple-Calculator#20]
Order of operations doesn't seem to work.
Example: 2+3*4 result is 20, but that's wrong. The correct result should be 14.

**Comment by tibbi** [comment]
the calculator is quite primitive and can have issues, and it even wont be my priority for some time as other apps perform better. So fixing things could take some time.

**Comment by WddVictor** [comment]
Actually, I don't think this is a bug. Because this is a simple calculator and the main purpose of this app is not to solve a complex equation like 1+3*2. It should display the result according to the order of the input. When you input the second operation *, the result should just base on the result of the previous operation. But if you do want to solve this, you can use `Stack` in the method handleCalculation to remember the order of input operations.

**Comment by Aga-C** [comment]
@tibbi Would you like to have this feature? Because, in my opinion, it should be simple with ExpressionBuilder that is used in the code.

**Comment by tibbi** [comment]
no, it isnt on the roadmap now, maybe someday in the scientific calculator

**Comment by ionuttbara** [comment]
there is a good calculator project which you can take note.
https://github.com/sadellie/unitto/releases/tag/nadeshiko-pink
so maybe someday simple calculator will be same or better than this app.

**Comment by inson1** [comment]
Without this is the app usless and not worth using

**Comment by inson1** [comment]
without it isnt the calculator worth using imho

**Comment by Novaliss** [comment]
Real human beings, their lives, safety and prosperity depends on not stumble upon a simple calculator that doesn't math. I mean, not a single user will double-check the output of an app that by its fundamentally simple nature, is expected to do the thing it says. The amount of various hazards implicated here are just staggering. You need to pull this trojan off the releases immediately until it is fixed.

**Comment by JimmysMaths** [comment]
@Novaliss  SimpleMobileTools apps were sold to ad and spyware company. 

Fork is here:
https://github.com/FossifyOrg

Discussion Here:
https://github.com/SimpleMobileTools/General-Discussion/issues/241
